api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -394,6 +394,7 @@ def get(self, key: str, default: Any | None = None) -> Any:
             if not self._obj.offererAddress:
                 return None
             offererAddress = GetOffererAddressResponseModel.from_orm(self._obj.offererAddress)
+            offererAddress.label = offererAddress.label if offererAddress.label else self._obj.venue.common_name
```

Comment:
S'écrit plus simplement : 
```suggestion
            offererAddress.label = offererAddress or self._obj.venue.common_name
```

Reply:
Merci 👍